osu.Game/OsuGame.cs

```diff
@@ -490,12 +477,13 @@ protected override void LoadComplete()
             dependencies.Cache(notifications);
             dependencies.Cache(dialogOverlay);
             dependencies.Cache(accountCreation);
+            dependencies.Cache(loginOverlay);
 
             chatOverlay.StateChanged += state => channelManager.HighPollRate.Value = state == Visibility.Visible;
 
             Add(externalLinkOpener = new ExternalLinkOpener());
 
-            var singleDisplaySideOverlays = new OverlayContainer[] { settings, notifications };
+            var singleDisplaySideOverlays = new OverlayContainer[] { settings, notifications, loginOverlay };
```

Comment:
this means that if you trigger a login popup from setting, the settings are going to disappear.  not sure this is what we want.

Reply:
If login and settings overlays are both open, the bottom overlay would be blocked (settings) with the current behavior of overlays.

With the dimming behind both of these overlays, both seem clickable (but the bottom overlay is blocked).